BrowserKit/Package.resolved

```diff
@@ -1,5 +1,14 @@
 {
   "pins" : [
```

Comment:
I don't think we should see those `Package.resolved` changes in this PR, can you have a look into that 👀 ?

Reply:
+1 on this comment, be sure to revert these changes. Also be sure to revert the changes in `version.xcconfig` that were made in this pr since they are unrelated to these changes.